CHANGES.rst

```diff
@@ -46,6 +46,28 @@ Release history
   as are commonly used to match biological spiking statistics. (`#1609`_)
 - Added the ``PositiveNeuronType`` test argument to run tests on all neuron types
   for which ``negative`` is not ``True``. (`#1609`_)
+- Added ``LinearSystem``, a ``Process`` to implement arbitrary linear systems.
+  (`#1611`_)
+- Added ``DoubleExp``, ``Bandpass``, ``Highpass``, ``DiscreteDelay``, and
+  ``LegendreDelay`` synapse models. (`#1611`_)
+- Added the ``ScatteredHypersphere`` distribution. This distribution is similar to
+  ``UniformHypersphere``, except that points are more evenly spread across the space
+  by use of quasirandom (i.e., low-discrepancy) number-theoretic sequences that minimize
+  the clusters and empty spaces that happen with independently drawn samples).
+  (`#1611`_)
+- Added the ``RLS`` (recursive least-squares) learning rule. This is an online version
+  of the least-squares method typically used for offline decoder-solving in Nengo.
+  It is demonstrated in the new ``learning/force-learning.ipynb`` notebook. (`#1611`_)
+- Added ``LinearSystemNetwork``. This network takes a ``LinearSystem`` process and
+  implements it in neurons, properly accounting for the synapse model used on the
+  recurrent synapse. For ``Lowpass`` synapses, this is equivalent to Principle 3 of the
+  Neural Engineering Framework (NEF), but it generalizes to other synapse types, as well
+  as discrete synapses. (`#1611`_)
+- Added ``rng`` argument to ``Process.make_state``, for processes that use randomness
```

Comment:
Should this be moved to changed, or should there otherwise be a note under changed, that processes written for v3.0.0 that override `make_state` need to be updated to work with v3.1.0? Or is this not a breaking change?